src/frformat/geo/code_commune_insee.py

```diff
@@ -1,9 +1,19 @@
-from frformat import enum_format
-from frformat.geo.code_commune_insee_set import CODE_COMMUNE_INSEE_SET
+from typing import Dict, Set
+
+from frformat import geo_enum_format
+from frformat.geo.code_commune_insee_set import (
+    CODE_COMMUNE_INSEE_SET_COG_2023,
+    CODE_COMMUNE_INSEE_SET_COG_2024,
+)
+from frformat.geo_enum_format import Millesime
 
 name = "Code commune INSEE"
-description = "Vérifie que le code commune correspond bien à un code commune INSEE"
+description = "Vérifie que le code commune correspond bien à un code commune INSEE selon Cpde Officiel géographique de 2023 et 2024"
```

Comment:
Petite coquille "Cpde" au lieu de "Code"

De manière générale, je ne préciserais pas quels codes sont disponibles pour ne pas avoir à le maintenir à chaque ajout. 

```suggestion
description = "Vérifie que le code commune correspond bien à un code commune INSEE pour un Code Officiel Géographique donné"
```